Frontend/src/components/DummyComponent/DummyComponent.jsx

```diff
@@ -1,11 +1,44 @@
 import React from 'react';
+import images from '../../constants/images';
 import styles from './DummyComponent.module.css';
 
 const DummyComponent = () => {
   return (
-    <div className={styles.headingContainer}>
-      <h1 className={styles.heading}>Welcome To BMC</h1>
-      <p className={styles.text}>We are glad to have you</p>
+    <div className={styles.bodyContainer}>
+      <div  className={styles.left}>
+        <div className="text-container">
+          <h1>TEMIBMC</h1>
+          <p>
+            Your business tool used to visualize all the building blocks when you want to start a business, including customers, route to market, value proposition and finance.
```

Comment:
**Update with:**

Temibmc is used to visualise all the building blocks of your business when starting a business, including customers, route to market, value proposition and finance.

Reply:
done